kolibri/utils/version.py

```diff
@@ -305,20 +335,16 @@ def get_prerelease_version(version):
 
     # Calculate suffix...
 
-    # If a description from git is available and we haven't already
-    # found a suffix
-    if not suffix and tag_describe:
+    # If a description from git is available and we are building on a commit
+    # with an exact tag
+    if git_tagged_version() and tag_describe:
```

Comment:
I think you can safely place the call to `git_tagged_version()` in  `tag_describe` above, so it becomes `tag_describe = git_tagged_version() or get_git_describe()`